raiden/raiden_service.py

```diff
@@ -534,7 +535,18 @@ def stop(self) -> None:
 
         log.debug("Raiden Service stopped", node=to_checksum_address(self.address))
 
-    def add_notification(self, notification: Notification) -> None:
+    def add_notification(
+        self,
+        notification: Notification,
+        log_opts: Optional[Dict] = None,
+        click_opts: Optional[Dict] = None,
```

Comment:
It would be nice if those could automatically be derived from the notification (e.g. from the severity).
But for now it's fine (see below).